code/obj/item/misc_weapons.dm

```diff
@@ -560,6 +575,14 @@ TYPEINFO(/obj/item/sword/pink/angel)
 		M.changeStatus("knockdown", 6 SECONDS)
 		M.force_laydown_standup()
 		take_bleeding_damage(M, null, 5, DAMAGE_CUT)
+		if (src.set_to_stick && istype(M, /mob/living))
+			var/mob/living/L = M
+			L.daggers += src
```

Comment:
This should probably come with some kind of visual indication that the dagger is stuck into someone. I know it's sprite work buuut it would be a lot clearer.

Reply:
Yeah was thinking, someone's going to be lying down, and depending on the way they are facing, the sprite may not even be seen at all, which is why I didn't sprite it

Reply:
I think it's still better than no indication, and you could probably fudge the sprites so it's visible in 3/4 directions anyway.